### DISCUSSION TITLE
How to get a rounded 2D shape into path_sweep()?

### DISCUSSION THREAD

**Discussion by dan-p3rry** [BelfrySCAD/BOSL2#1497] [Q&A] [answered]
The documentation for path_sweep() shows several examples of sweeping a U shape, where the U shape is formed by listing out the coordinates.  Is there a way to round the corners of the U shape before sweeping?

**Comment by revarbat** [comment, marked as answer]
Off the top of my head, I think you need `round_corners()` from rounding.scad

**Reply by dan-p3rry** [reply]
Yes, that worked, thanks.  I had been trying round2d(), but that's a module, not a function.
